test/test_reductions.py

```diff
@@ -416,6 +416,20 @@ def test_sum_integer_upcast(self, device):
         self._test_reduce_integer_upcast(lambda x, **kwargs: torch.sum(x, **kwargs), False)
         self._test_reduce_integer_upcast(lambda x, **kwargs: torch.sum(x, 0, **kwargs))
 
+    @onlyCPU
+    def test_cumsum_bfloat16(self, device):
```

Comment:
There's also a test_cumsum in test_torch.py. Could you add BFloat16 dtype to those tests?

Reply:
sure, I moved the BFloat16 test of cumsum to test_cumsum in test_torch.py.